### ISSUE TITLE
@SUMMONENVFILE does not include trailing newline

### ISSUE THREAD

**Issue by chrisdostert** [conjurinc/summon#22]
it looks like the generated @SUMMONENVFILE does not include a newline character on the last line. This leads to shell utilities adhering to POSIX standards ignoring the last line when reading the file.

**Comment by dustinmm80** [comment]
Interesting, does your input secrets.yml file have a trailing newline? I haven't run into this issue yet

**Comment by chrisdostert** [comment]
Yes, my secrets.yml does have a trailing newline character.

A quick repro:
```SHELL
summon while IFS= read -r line; do echo $line; done < @SUMMONENVFILE
# last secret in @SUMMONENVFILE will not be output due to missing trailing newline char
```

**Comment by chrisdostert** [comment]
I can't repro all of a sudden.. don't know if it was me or gremlins.. i'll keep an eye on it and report back if It resurfaces and I am able to track it down farther. Thanks and sorry to waste time.

**Comment by chrisdostert** [comment]
arg.. was me having a "moment". It does still repro every time.

**Comment by dustinmm80** [comment]
This issue has been resolved in v0.3.2 - https://github.com/conjurinc/summon/releases/tag/v0.3.2
